blackbox/docs/appendices/release-notes/3.3.0.rst

```diff
@@ -0,0 +1,163 @@
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+.. _inserting the data into a new table: https://crate.io/docs/crate/reference/en/latest/admin/system-information.html#tables-need-to-be-recreated
+
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+.. _version_3.3.0_upgrade_notes:
+
+Upgrade Notes
+=============
+
+Deprecated Settings and Features
+--------------------------------
+
+The query frequency and average duration :ref:`query_stats_mbean` metrics
+has now been deprecated, in favour of the new total count and sum of durations
+metrics.
+
+The :ref:`cluster.graceful_stop.reallocate <cluster.graceful_stop.reallocate>`
+setting has been marked as deprecated. This setting was already being ignored,
+and setting the value to `false` had no effect.
+
+The node decommission using the ``USR2`` :ref:`cli_signals` has now been
+deprecated in favour of the
+:ref:`ALTER CLUSTER DECOMISSION <alter_cluster_decommission>` statement.
+
+The :ref:`create-ingest-rule` and :ref:`drop-ingest-rule` rules have been marked
+as deprecated. Given that the only implementation (MQTT) was deprecated
+and will be removed, the framework itself will also be removed.
+
+
+Changelog
+=========
+
+Changes
+-------
+
+New Features
+~~~~~~~~~~~~
+
+- Exposed the sum of durations, total, and failed count metrics under the
+  :ref:`query_stats_mbean` for ``QUERY``, ``INSERT``, ``UPDATE``, ``DELETE``,
+  ``MANAGEMENT``, ``DDL`` and ``COPY`` statement types.
+
+- Exposed the sum of statement durations, total, and failed count classified by
+  statement type under the sum_of_durations, total_count and failed_count
+  columns, respectively, in the :ref:`sys-jobs-metrics` table.
+
+SQL Improvements
+~~~~~~~~~~~~~~~~
+
+- Added ``current_schemas(boolean)`` scalar function which will return the
+  names of schemas in the ``search_path``.
+
+- Added support for the ``first_value``, ``last_value`` and ``nth_value``
+  window functions as enterprise features.
+
+- Added the ``DROP ANALYZER`` statement to support removal of custom
+  analyzer definitions from the cluster.
+
+- Output the custom analyzer/tokenizer/token_filter/char_filter definition inside
+  the ``information_schema.routines.routine_definition`` column.
+
+- Added support for the window function ``row_number()``.
+
+- Added support to use any expression in the operand of a ``CASE`` clause.
+
+PostgreSQL Compatibility
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Added ``pg_type`` columns: ``typlen``, ``typarray``, ``typnotnull``
+  and ``typnamespace`` for improved postgresql compatibility.
+
+- Added a ``pg_description`` table to the ``pg_catalog`` schema for improved
+  postgresql compatibility.
+
+- Fixed function resolution for postgresql functions ``pg_backend_pid``,
+  ``pg_get_expr`` and ``current_database`` when the schema prefix
+  ``pg_catalog`` is included.
+
+Database Administration
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Added a node check that checks the JVM version under which CrateDB is
```

Comment:
"a node check for the JVM version number." (drop everything after in this sentence)